models/datacenter.go

```diff
@@ -35,80 +35,80 @@ import (
 type Datacenter struct {
 
 	// The administrative state of the resource, which is up (true) or down (false). Default is true.
-	AdminStateUp *bool `json:"admin_state_up,omitempty" db:"admin_state_up,omitempty"`
+	AdminStateUp *bool `json:"admin_state_up,omitempty"`
```

Comment:
what happened here (and everywhere else in the models) is that the struct-tags db has been removed because go-swagger was invoked without the required parameters (e.g. `--struct-tags db`), Please use `make swagger` which will set the correct swagger options.